docs/manual/meta/scene_manager.rst

```diff
@@ -0,0 +1,112 @@
+
+.. image:: img/spatial_entities.png
+
+First create a new scene with the type of ``StaticBody3D`` and add a ``Label3D`` to it. The scene should now look like this:
+
+.. image:: img/spatial_entity_scene.png
+
+Next, add a new script with the following content:
+
+.. code-block:: gdscript
+
+    extends StaticBody3D
+
+    @onready var label: Label3D = $Label3D
+
+    func setup_scene(entity: OpenXRFbSpatialEntity) -> void:
+        var semantic_labels: PackedStringArray = entity.get_semantic_labels()
+
+        label.text = semantic_labels[0]
+
+        var collision_shape = entity.create_collision_shape()
+        if collision_shape:
+            add_child(collision_shape)
+
+        var mesh_instance = entity.create_mesh_instance()
+        if mesh_instance:
+            add_child(mesh_instance)
+
+This script will create a new create the collision shape and mesh instance for the StaticBody3D. The :ref:`create_collision_shape() <class_openxrfbspatialentity_method_create_collision_shape>` creates a new collision shape and the :ref:`create_mesh_instance() <class_openxrfbspatialentity_method_create_mesh_instance>` creates a new mesh instance. Both functions return a new node that can be added to the scene.
+Alternatively you can use the :ref:`get_triangle_mesh() <class_openxrfbspatialentity_method_get_triangle_mesh>` function to get the mesh data and create your own mesh instance and collision shape.
+
+Additionally, the :ref:`get_semantic_labels() <class_openxrfbspatialentity_method_get_semantic_labels>` function returns a list of labels that are associated with the spatial entity. Normally this list will only contain one label.
+
+Now add the scene to the Scene Manager by setting the **Default Scene** option or any of the options in the **Scenes** section to the newly created scene.
+
+Global Mesh
+-----------
+
+The Global Mesh is created by the Headset when you scan your space with the depth sensor. The Global Mesh can be used for letting physics objects collide with the real world or to occlude virtual objects behind static physical objects.
+
+|global_mesh| |global_mesh_wireframe|
+
+.. |global_mesh| image:: img/global_mesh.png
+    :width: 49%
+.. |global_mesh_wireframe| image:: img/global_mesh_wireframe.png
+    :width: 49%
+
+The process of creating the the template scene for the global mesh is the same as for any other spatial entity.
+
+Requesting a Scene Capture
+--------------------------
+
+In case the space hasn't been set up yet, you can use the :ref:`request_scene_capture() <class_openxrfbscenemanager_method_request_scene_capture>` method to request a scene capture.
+The app will be paused and the user will be asked to scan the room their in.
+
+The :ref:`on_open_xr_fb_scene_capture_completed(success: bool) <class_openxrfbscenemanager_signal_openxr_fb_scene_capture_completed>` signal can be used to act on the completion of the scene capture.
```

Comment:
Getting VERY nitpicky here but do we have a way we'd like to format linking to methods/signals? For some reason having the `success: bool` in there initially struck me as a typo. Do the godot docs link to signals/methods like this?

Reply:
Will remove it for consistency. The thought behind it was that it would be more transparent what the signal does but it makes updating the params really annoying as these dependencies wouldn't update with it.